roles/StackStorm.nodejs/tasks/nodejs_redhat.yml

```
@@ -19,6 +19,28 @@
   register: nodesource_repo_rm
   tags: nodejs
 
+- name: Determine if nodejs disabled on AppStream for RH8
```

Super picky - but let use EL8?

```suggestion
- name: Determine if nodejs disabled on AppStream for EL8
```